lua/ulx/modules/sh/cfc_owoify.lua

```diff
@@ -0,0 +1,44 @@
+local CATEGORY_TYPE = "Chat"
+local OWOIFYDICT = { ove = "uv", na = "nya", ne = "nye", ni = "nyi", no = "nyo", nu = "nyu", r = "w", l = "w", O = "owo", U = "uwu", wh = "w" }
+local owoifyTargets = {}
+
+local function owoifyString( inputStr )
+    for k, v in pairs( OWOIFYDICT ) do
+        inputStr = string.Replace( inputStr, k, v )
+    end
+
+    return inputStr
+end
+
+local function onPlayerSay( ply, inputStr )
+    if not table.HasValue( owoifyTargets, ply ) then return end
+
+    return owoifyString( inputStr )
+end
+
+local function owoifyOn( caller, targets )
+    table.Add( owoifyTargets, targets )
+    hook.Add( "PlayerSay", "CFC_ULX_OwoifyString", onPlayerSay )
```

Comment:
I appreciate your tidiness, cleaning up the hook when you don't need it and adding it when you do. There's absolutely nothing wrong with that, I even do it in some places.

It does add some complexity and might be a little weird to debug later ("wait where's my hook?"), so that's something to consider.

Personally, I wouldn't bother in this case, but there's no reason to undo it 👍